specifyweb/frontend/js_src/lib/components/Merging/InvalidMergeRecords.tsx

```diff
@@ -0,0 +1,82 @@
+import { RA } from '../../utils/types';
+import { AnySchema, SerializedResource } from '../DataModel/helperTypes';
+import { H2, Ul } from '../Atoms';
+import { FormattedResource } from '../Molecules/FormattedResource';
+import { deserializeResource } from '../DataModel/helpers';
+import React from 'react';
+import { OverlayContext } from '../Router/Router';
+import { Dialog } from '../Molecules/Dialog';
+import { recordMergingTableSpec } from './definitions';
+import { dialogIcons } from '../Atoms/Icons';
+import { Button } from '../Atoms/Button';
+import { commonText } from '../../localization/common';
+import { mergingText } from '../../localization/merging';
+import { Tables } from '../DataModel/types';
+import { TableIcon } from '../Molecules/TableIcon';
+
+function InvalidMergeRecords({
+  resources,
+  header = '',
+}: {
+  readonly resources: RA<SerializedResource<AnySchema>>;
+  readonly header?: string;
+}): JSX.Element {
+  return (
+    <div className="flex flex-1 flex-col gap-4">
+      <H2>{header}</H2>
+      <Ul className="gap-2">
+        {resources.map((resource, index) => (
+          <li
+            key={(resource.id as number) ?? index}
+            className="flex flex min-h-[theme(spacing.8)] flex-1 items-center gap-2"
+          >
+            <TableIcon label={true} name={resource._tableName} />
```

Comment:
since resource is SerializedResource and not SpecifyResource, `resource.specifyModel.name` won't work.

Good point about it always being defined. I just passed it down for sanity instead